packages/e2e-test-utils-playwright/src/request-utils/comments.ts

```
@@ -7,20 +7,35 @@ export interface Comment {
 	id: number;
 	author: number;
 	content: string;
+	post: number;
 }
 
 /**
  * Create new comment using the REST API.
  *
- * @param {} this    RequestUtils.
- * @param {} comment Comment.
+ * It uses the current user if no author is defined inside `comment`.
+ *
+ * @param {} this           RequestUtils.
+ * @param {} comment        Partial<Comment>.
  */
-export async function createComment( this: RequestUtils, comment: Comment ) {
-	this.rest( {
+export async function createComment(
+	this: RequestUtils,
+	comment: Partial< Comment >
+) {
+	const currentUser = await this.rest( {
+		path: '/wp/v2/users/me',
+		method: 'GET',
+	} );
+
+	const author = currentUser?.id;
```

Nit: We might want to throw an error early if `currentUser` is `null` or `undefined`, but I don't think that would happen? Let's remove the premature optional chaining so that it doesn't hide where the error really is.